### ISSUE TITLE
i2p.bote.folder.Folder : Can't create a FolderElement from file (Trac #937)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#14]
ERROR [RelayPktSndr] i2p.bote.folder.Folder : Can't create a FolderElement from file: /var/lib/i2p/i2p-config/i2pbote/relay_pkt/[REDACTED].pkt (file size=0)
     java.lang.NullPointerException
     at i2p.bote.folder.RelayPacketFolder.createFolderElement(RelayPacketFolder.java:79)
     at i2p.bote.folder.RelayPacketFolder.createFolderElement(RelayPacketFolder.java:39)
     at i2p.bote.folder.Folder$3.findNextElement(Folder.java:164)
     at i2p.bote.folder.Folder$3.next(Folder.java:145)
     at i2p.bote.folder.PacketFolder$1.next(PacketFolder.java:106)
     at i2p.bote.folder.PacketFolder$1.next(PacketFolder.java:90)
     at i2p.bote.service.RelayPacketSender.run(RelayPacketSender.java:69)

2 times this error

Migrated from https://trac.i2p2.de/ticket/937
```json
{
    "status": "closed", 
    "changetime": "2015-07-22T21:17:05", 
    "description": "ERROR [RelayPktSndr] i2p.bote.folder.Folder : Can't create a FolderElement from file: /var/lib/i2p/i2p-config/i2pbote/relay_pkt/[REDACTED].pkt (file size=0)\n     java.lang.NullPointerException\n     at i2p.bote.folder.RelayPacketFolder.createFolderElement(RelayPacketFolder.java:79)\n     at i2p.bote.folder.RelayPacketFolder.createFolderElement(RelayPacketFolder.java:39)\n     at i2p.bote.folder.Folder$3.findNextElement(Folder.java:164)\n     at i2p.bote.folder.Folder$3.next(Folder.java:145)\n     at i2p.bote.folder.PacketFolder$1.next(PacketFolder.java:106)\n     at i2p.bote.folder.PacketFolder$1.next(PacketFolder.java:90)\n     at i2p.bote.service.RelayPacketSender.run(RelayPacketSender.java:69)\n\n2 times this error", 
    "reporter": "guest", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1437599825195621", 
    "component": "apps/plugins", 
    "summary": "i2p.bote.folder.Folder : Can't create a FolderElement from file", 
    "priority": "minor", 
    "keywords": "I2P-Bote", 
    "version": "0.9.5", 
    "parents": "", 
    "time": "2013-05-26T19:44:18", 
    "milestone": "0.9.21", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20130527T00:36:55: zzz changed component from "other" to "apps/plugins"

**Comment by str4d** [comment]
Trac update at 20130527T00:37:19: 
* zzz changed owner from "" to "HungryHobo"
* zzz changed status from "new" to "assigned"

**Comment by str4d** [comment]
Trac update at 20131225T15:57:32: 
* str4d changed keywords from "" to "i2pbote"
* str4d changed milestone from "0.9.7" to ""
* str4d changed parents from "" to ""

**Comment by str4d** [comment]
Trac update at 20140327T23:20:26: str4d commented:

> Possibly related: #16

**Comment by str4d** [comment]
Trac update at 20150110T04:00:53: str4d changed keywords from "i2pbote" to "I2P-Bote"

**Comment by str4d** [comment]
Trac update at 20150722T21:16:38: 
* str4d changed owner from "HungryHobo" to "str4d"
* str4d changed status from "assigned" to "accepted"

**Comment by str4d** [comment]
Trac update at 20150722T21:17:05: 
* str4d commented:

> Fixed in `8f97cdacfdb3a1c3cb25512a0f55433174d554a5`.


* str4d changed milestone from "" to "0.9.21"
* str4d changed resolution from "" to "fixed"
* str4d changed status from "accepted" to "closed"
